applications/crossbar/src/modules/cb_agents.erl

```diff
@@ -309,6 +312,35 @@ fetch_all_current_stats(Context, AgentId) ->
             ]),
     fetch_stats_from_amqp(Context, Req).
 
+-spec fetch_current_status(cb_context:context(), api_binary(), api_boolean()) -> cb_context:context().
+fetch_current_status(Context, AgentId, Full) ->
+    case Full of
```

Comment:
Use 2 function clauses instead of 2 case clauses please:

```erlang
fetch_current_status(Context, AgentId, 'false') ->
    …;
fetch_current_status(Context, AgentId, 'true') ->
    ….
```